tripy/tripy/frontend/module/layernorm.py

```diff
@@ -92,9 +104,18 @@ def __call__(self, x: "tripy.Tensor") -> "tripy.Tensor":
             A tensor of the same shape as the input.
         """
         from tripy.frontend.trace.ops.reduce import mean, var
+        from tripy.frontend.shape import Shape
         from tripy.frontend.trace.ops.unary_elementwise import rsqrt
+        from tripy.common.exception import raise_error
+
+        # The mean and the variance are computed over the last D dimensions
+        D = len(self.normalized_shape)
+
+        if x.shape[-D:] != Shape(self.normalized_shape):
```

Comment:
I thought we're able to compare shape tensors directly against lists? Can we avoid the explicit constructor here?